src/SumoCoders/FrameworkCoreBundle/Form/Type/AbstractImageType.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace SumoCoders\FrameworkCoreBundle\Form\Type;
+
+use Symfony\Component\Form\AbstractType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\Form\FormInterface;
+use Symfony\Component\Form\FormView;
+use Symfony\Component\OptionsResolver\OptionsResolverInterface;
+
+class AbstractImageType extends AbstractType
```

Comment:
fair point, I'll refactor this :) this is the default implementation but since there are no abstract methods it can just be a class that gets extended